crates/nu-command/src/filesystem/mktemp.rs

```diff
@@ -0,0 +1,133 @@
+use nu_engine::env::current_dir;
+use nu_engine::CallExt;
+use nu_protocol::ast::Call;
+use nu_protocol::engine::{Command, EngineState, Stack};
+use nu_protocol::{
+    Category, Example, PipelineData, ShellError, Signature, Span, Spanned, SyntaxShape, Type, Value,
+};
+
+use std::path::PathBuf;
+
+#[derive(Clone)]
+pub struct Mktemp;
+
+impl Command for Mktemp {
+    fn name(&self) -> &str {
+        "mktemp"
+    }
+
+    fn usage(&self) -> &str {
+        "Create temporary files or directories using uutils/coreutils mktemp. TEMPLATE must contain at least 3 consecutive 'X's in last component."
```

Comment:
```suggestion
        "Create temporary files or directories using uutils/coreutils mktemp."
```
That template suggestion is already in the help text so I think we can get rid of it here.